framework/test/unit/modules/auth/endpoint.spec.ts

```diff
@@ -487,4 +487,52 @@ describe('AuthEndpoint', () => {
 			);
 		});
 	});
+
+	describe('sortMultisignatureGroup', () => {
+		it('should sort signatures when provided mandatory and optional keys', () => {
+			// Arrange
+			const inputData = {
+				mandatory: [
+					{
+						publicKey: '3333333333333333333333333333333333333333333333333333333333333333',
+						signature:
+							'22222222222222222222222222222222222222222222222222222222222222222222222222222222222222222222222222222222222222222222222222222222',
+					},
+					{
+						publicKey: '0000000000000000000000000000000000000000000000000000000000000000',
+						signature:
+							'11111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111',
+					},
+					{
+						publicKey: 'bbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbb',
+						signature:
+							'00000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000',
```

Comment:
I think it would be nice to have empty signature case here as well even though it will be handled in the implementation correctly

Reply:
Scroll only 6 lines down to the line 515. There is the empty signature test data already 🙂

It doesn't make any difference at all if signature is empty or not. The sorting logic uses only `publicKey` property. It will correctly assign the relevant value of `signature` property to the response array, regardless of if the signature is empty or not.

Reply:
ah yea true the test was there! my bad =)